src/Illuminate/Foundation/Console/ShowModelCommand.php

```diff
@@ -0,0 +1,433 @@
+            default => $value,
+        };
+    }
+
+    /**
+     * Get the cast type for the given column.
+     *
+     * @param  string  $column
+     * @param  \Illuminate\Database\Eloquent\Model  $model
+     * @return string|null
+     */
+    protected function getCastType($column, $model)
+    {
+        if ($model->hasGetMutator($column) || $model->hasSetMutator($column)) {
+            return 'Accessor';
+        }
+
+        if ($model->hasAttributeMutator($column)) {
+            return 'Attribute';
+        }
+
+        return $this->getCastsWithDates($model)->get($column) ?? null;
+    }
+
+    /**
+     * Get the model casts, including any date casts.
+     *
+     * @param  \Illuminate\Database\Eloquent\Model  $model
+     * @return Collection
+     */
+    protected function getCastsWithDates($model)
+    {
+        return collect([
+            ...collect($model->getDates())->flip()->map(fn () => 'datetime'),
+            ...$model->getCasts(),
+        ]);
+    }
+
+    /**
+     * Get the type of the given column.
+     *
+     * @param  \Doctrine\DBAL\Schema\Column  $column
+     * @return string
+     */
+    protected function getColumnType($column)
+    {
+        $name = $column->getType()->getName();
+
+        $details = match (get_class($column->getType())) {
+            DecimalType::class => $column->getPrecision().','.$column->getScale(),
+            default => $column->getLength(),
+        };
+
+        $unsigned = $column->getUnsigned() ? ' unsigned' : '';
+
+        if ($details) {
+            return sprintf('%s(%s)%s', $name, $details, $unsigned);
+        }
+
+        return sprintf('%s%s', $name, $unsigned);
+    }
+
+    /**
+     * Get the default value for the given column.
+     *
+     * @param  \Doctrine\DBAL\Schema\Column  $column
+     * @param  \Illuminate\Database\Eloquent\Model  $model
+     * @return string|null
+     */
+    protected function getColumnDefault($column, $model)
+    {
+        if ($column->getAutoincrement()) {
+            return 'increments';
+        }
+
+        return $model->getAttributes()[$column->getName()] ?? $column->getDefault();
+    }
+
+    /**
+     * Determine if the given attribute is hidden.
+     *
+     * @param  string  $attribute
+     * @param  \Illuminate\Database\Eloquent\Model  $model
+     * @return bool
+     */
+    protected function attributeIsHidden($attribute, $model)
+    {
+        if (count($model->getHidden()) > 0) {
+            return in_array($attribute, $model->getHidden());
+        }
+
+        if (count($model->getVisible()) > 0) {
+            return ! in_array($attribute, $model->getVisible());
+        }
+
+        return false;
+    }
+
+    /**
+     * Qualify the given model class base name.
+     *
+     * @param  string  $model
+     * @return string
+     *
+     * @see \Illuminate\Console\GeneratorCommand
+     */
+    protected function qualifyModel(string $model)
+    {
+        $model = ltrim($model, '\\/');
```

Comment:
```suggestion
        if (class_exists($model)) {
            return $model;
        }
        
        $model = ltrim($model, '\\/');
```

Currently, it's not possible to use the command when not following Laravel's default architecture - using a domain-driven approach, models are not necessarily in `App/Models`. 

With this condition, I can do `php artisan model:show Domain\\Reporting\\Models\\Duty`

Reply:
Makes sense!

This method was taken pretty much verbatim from the `make:*` commands so I'm assuming they're similarly affected?

https://github.com/laravel/framework/blob/77560968f6a308d2abddb832ad733bee2b15392e/src/Illuminate/Console/GeneratorCommand.php#L207-L228

Reply:
They are, unfortunately :(

With a DDD architecture, `make:*` commands are not really usable - we have to move files and update namespaces manually